doc-content/enterprise-only/installation/clustering-bc-configure-standalone_proc.adoc

```diff
@@ -39,35 +45,43 @@ These steps describe a basic cluster configuration. For more complex configurati
      value="<INFINISPAN_NODE_IP>"/>
   <property name="org.appformer.ext.metadata.infinispan.realm"
      value="ApplicationRealm"/>
-  <property name="org.appformer.ext.metadata.infinispan.server.name"
-     value="<SERVER_NAME>"/>
-  <property name="org.appformer.ext.metadata.infinispan.sasl.qop"
-     value="<SASL_QOP>"/>
-  <property name="org.appformer.ext.metadata.infinispan.username"
-     value=""/>
-  <property name="org.appformer.ext.metadata.infinispan.password"
-     value=""/>
   <property name="org.appformer.ext.metadata.infinispan.cluster"
      value="kie-cluster"/>
   <property name="org.appformer.ext.metadata.index"
      value="infinispan"/>
   <property name="org.uberfire.nio.git.dir"
-     value="<SHARED_NETWORK_FOLDER>"/>
+     value="/data"/>
   <property name="es.set.netty.runtime.available.processors"
      value="false"/>
 </system-properties>
 ----
++
+. If the {DATAGRID} deployment requires authencication, also edit or add the following properties under the `<system-properties>` element and replace the following placeholders:
```

Comment:
Suggestions - change step to optional, correct typo, remove "also", and minor rephrase. See below:

. Optional: If the {DATAGRID} deployment requires authentication, edit or add the properties under the `<system-properties>` element and replace the following placeholders: